### PR TITLE
Reviewer changes to HLP-518

### PR DESCRIPTION
**Pull request by sfoks**
# Pull Request Code Review

## Overview

I made some reviewer changes to HLP-518 on the visualization notebook for CONUS404 evaluation.

## Links

- This PR is the reviewer changes for [this PR](https://github.com/hytest-org/hytest/pull/560).

## Summary of Changes

-  I removed the section on DRY principles because I felt it could go elsewhere and didn't need to be in this particular notebook (maybe in the future in the essential reading section).
- I changed the first gridded plot to map the same date for comparison.
- I changed the second plot to highlight the station that is selected for the timeseries.
- I added a section for examining the evaluation results of the station data vs CONUS404 (just precip and temp). 

## How to Run

Run as a normal notebook.

## Peer Reviewer Expectations

- [x] Please check that it runs, I didn't misspell anything, and that you are okay with what I changed before we merge this into HLP-518 and then main.

### CODE REVIEW

File: evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb [hunk 1]
```diff
@@ -1030,14 +166,14 @@
    "source": [
     "## **Step 2: Visualize and Compare Gridded Data**\n",
     "\n",
-    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset ('conus404-drb-OSN') and the PRISM Delaware River Basin subset ('prism-drb-OSN'). For the purposes of this notebook, we will only visualize the most recent data in each dataset.\n",
+    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset (_'conus404-drb-OSN'_) and the PRISM Delaware River Basin subset (_'prism-drb-OSN'_). For the purposes of this notebook, we will only visualize the most recent data in each dataset.\n",
     "\n",
-    "Let's start by loading in our two datasets and combining them into a list."
+    "Let's start by loading in our two datasets and combining them into a list. Note that the timesteps in these datasets are the same (monthly) though have different periods of record, we will use the first time point for comparison."
```
Comment (hpodzorski-USGS) [inline evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb:171]:
```suggestion
    "Let's start by loading in our two datasets and combining them into a list. Note that the timesteps in these datasets are the same (monthly). However, they have different periods of record. We will use the first time point for comparison."
```

Comment (hpodzorski-USGS) [reply evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb:171]: I was having trouble understanding this sentence and suggested a rewrite to help with clarity.

File: evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb [hunk 2]
```diff
@@ -1030,14 +166,14 @@
    "source": [
     "## **Step 2: Visualize and Compare Gridded Data**\n",
     "\n",
-    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset ('conus404-drb-OSN') and the PRISM Delaware River Basin subset ('prism-drb-OSN'). For the purposes of this notebook, we will only visualize the most recent data in each dataset.\n",
+    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset (_'conus404-drb-OSN'_) and the PRISM Delaware River Basin subset (_'prism-drb-OSN'_). For the purposes of this notebook, we will only visualize the most recent data in each dataset.\n",
```
Comment (hpodzorski-USGS) [inline evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb:169]:
```suggestion
    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset (_'conus404-drb-OSN'_) and the PRISM Delaware River Basin subset (_'prism-drb-OSN'_).\n",
```

File: evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb (hunk 1 above)
Comment (sfoks) [reply evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb:171]: This sounds better, I'll make this change.

**Pull request merged**